src/execution/sql/ddl_executors.cpp

```diff
@@ -79,7 +79,7 @@ bool DDLExecutors::CreateTableExecutor(const common::ManagedPointer<planner::Cre
     for (const auto &unique_col : unique_constraint.unique_cols_) {
       const auto &table_col = schema.GetColumn(unique_col);
       if (table_col.Type() == type::TypeId::VARCHAR || table_col.Type() == type::TypeId::VARBINARY) {
```

Comment:
Probably need decimal here too.

Reply:
I think we punt that type of change until the end-to-end DECIMAL support PR, especially if you've made those changes already. I wouldn't be able to exercise them yet.